docs/management_api.yml

```diff
@@ -1183,6 +1183,30 @@ paths:
         500:
           $ref: "#/responses/InternalServerError"
 
+  /artifacts/directupload:
+    post:
+      operationId: Request Direct Upload
+      tags:
+        - Management API
+      security:
+        - ManagementJWT: []
+      summary: Request link for uploading artifact directly to the storage backend
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+            properties:
+              $ref: "#/definitions/ArtifactUploadLink"
+        400:
+          $ref: "#/responses/InvalidRequestError"
```

Comment:
idle comment: every time I read it I have the same thought (usually unexpressed): 400 is a _Bad Request_ and should be addressed like that, I checked the desc of this ref, it is not right.